instruction/programming-exam/programming-exam.md

```diff
@@ -0,0 +1,164 @@
+
+The practice quizzes and the actual programming exam are available on Canvas in the "Programming Exam" section of the "Assignments" tab.
+
+## Pre Exam Setup
+
+Make sure that you take time to do all of these steps before you start the actual exam, as they will take some time, and can have large problems come up during setup. If you want help to do these steps, or have questions about what is allowed before you start, talk with a TA.
+
+Before taking the exam you should complete all of the steps in the chess phase-0 [getting started](https://github.com/softwareconstruction240/softwareconstruction/blob/main/chess/0-chess-moves/getting-started.md) document, except you will not put the project in a GitHub repo. Just create a folder and set it up with the starter code, and open it in Intellij. This will mean you will have an Intellij project with the provided starter code. You should make sure the project is able to compile and the tests run (but still all fail) before starting. Do not add any code, psuedocode, or notes to your project anywhere, this will be treated as cheating.
+
+You should also open the Javadoc in a web browser tab and the [chess phase-0 specification](https://github.com/softwareconstruction240/softwareconstruction/blob/main/chess/0-chess-moves/chess-moves.md) in another browser tab. If you are taking the test during TA hours you can also have the help queue open. If you need to talk with a TA you can open Zoom for that purpose. All other programs should be closed so that you have only the approved programs open on your computer during the exam. .
+
+You may not open other tabs or access other information during the exam. This is designed to truly test that you have the capability to program the project.
+
+If you are going to make handwritten notes while working have a blank sheet of paper ready to show to the camera at the start of the exam.
+
+When planning your time out allow yourself at least 20 minutes at the end of the exam to submit everything.
+
+You will need to upload:
+- A zip of your entire project
+- A screenshot of all your tests passing (right click passoff, click “run tests in passoff”). There should be 46 tests passing. If you don't have this number you are not running all of the tests.
+
+Make sure you know how to do both of these things before the exam starts so you don't lose time.
+
+
+## Warnings and Other Information
+
+**Please read this section carefully**
+
+Once you start the exam, do not close any web browser tabs. Sometimes closing a tab causes Examity to terminate the exam.
+
+You may take the exam at any time during the exam period. Just make sure you give yourself enough time to complete the exam by the end of the exam period.
+
+⚠ The exam is not open until midnight the day of the exam. It closes at 5, so make sure that you start with enough time.
```

Comment:
Having a different ending time from all the other assignments has caused significant confusion and frustration for many learners. We are considering changing this to midnight and this reference will create a duplication of the deadline that might be incorrect.